### DISCUSSION TITLE
In what situations do AdGuard Family DNS server use Safe Search?

### DISCUSSION THREAD

**Discussion by ghost** [AdguardTeam/AdGuardHome#4979] [Q&A]
If I use AdGuard Family DNS servers without enabling Safe Search in AGH settings, then is Safe Search used?

What about setting AdGuard Family DNS server in router settings? Would that force Safe Search? I want the familly level of filtering, but don't want to send any info to AdGuard.

**Comment by ainar-g** [comment]
Yes, AdGuard DNS Family Protection servers have the Safe Search feature enabled.
